src/mame/seta/seta.cpp

```diff
@@ -10227,97 +10254,97 @@ ROM_START( jjsquawko ) /* Official 93111A PCB missing version sticker */
 ROM_END
 
 ROM_START( jjsquawkb )
-	ROM_REGION( 0x200000, "maincpu", 0 )        /* 68000 Code */
+	ROM_REGION( 0x200000, "maincpu", 0 )        // 68000 Code
 	ROM_LOAD16_WORD_SWAP( "3", 0x000000, 0x080000, CRC(afd5bd07) SHA1(eee231f596ce5cb9bbf41c7c9e18c11a399d7dfd) )
 	ROM_LOAD16_WORD_SWAP( "2", 0x100000, 0x080000, CRC(740a7366) SHA1(2539f9a9b4fed1a1e2c354d144b8d455ed4bc144) )
 
-	ROM_REGION( 0x400000, "gfx1", 0 )   /* Sprites */
+	ROM_REGION( 0x400000, "gfx1", 0 )   // Sprites
 	ROM_LOAD( "4.bin",  0x000000, 0x200000, CRC(969502f7) SHA1(d6cecb38e8b73c61537cc1bdc843fc7cd695c771) ) // sldh
 	ROM_LOAD( "2.bin",  0x200000, 0x200000, CRC(765253d1) SHA1(4cbc6f093c87280ef9c17fecfc319cb780d755cc) ) // sldh
 
 	ROM_REGION( 0x400000, "gfxtemp", 0  )
 	ROM_LOAD( "3.bin",  0x000000, 0x200000, CRC(b1e3a4bb) SHA1(be2241a4fbb99444487e7b550faac4ee1ee1ad15) ) // sldh
 	ROM_LOAD( "1.bin",  0x200000, 0x200000, CRC(a5d37cf7) SHA1(9573777f3cdd6b25f0bd56f65f583fddda21c900) ) // sldh
 
-	ROM_REGION( 0x200000, "gfx2", 0 )   /* Layer 1 */
+	ROM_REGION( 0x200000, "gfx2", 0 )   // Layer 1
 	ROM_COPY( "gfxtemp", 0x000000, 0x000000, 0x100000 )
 	ROM_COPY( "gfxtemp", 0x200000, 0x100000, 0x100000 )
 
-	ROM_REGION( 0x200000, "gfx3", 0 )   /* Layer 2 */
+	ROM_REGION( 0x200000, "gfx3", 0 )   // Layer 2
 	ROM_COPY( "gfxtemp", 0x100000, 0x000000, 0x100000 )
 	ROM_COPY( "gfxtemp", 0x300000, 0x100000, 0x100000 )
 
-	ROM_REGION( 0x100000, "x1snd", 0 )  /* Samples */
-	ROM_LOAD( "1", 0x000000, 0x100000, CRC(181a55b8) SHA1(6fa404f85bad93cc15e80feb61d19bed84602b82) ) /* fe2001005.u69 + fe2001006.u70 from jjsquawk */
+	ROM_REGION( 0x100000, "x1snd", 0 )  // Samples
+	ROM_LOAD( "1", 0x000000, 0x100000, CRC(181a55b8) SHA1(6fa404f85bad93cc15e80feb61d19bed84602b82) ) // fe2001005.u69 + fe2001006.u70 from jjsquawk
 ROM_END
 
-ROM_START( jjsquawkb2 ) /* PCB was P0-078A, which was a Blandia board converted to JJ Squawkers. No labels on any of the ROMs.  Apparently based on jjsquawko set. */
-	ROM_REGION( 0x200000, "maincpu", 0 )        /* 68000 Code */
+ROM_START( jjsquawkb2 ) // PCB was P0-078A, which was a Blandia board converted to JJ Squawkers. No labels on any of the ROMs. Apparently based on jjsquawko set.
+	ROM_REGION( 0x200000, "maincpu", 0 )        // 68000 Code
```

Comment:
Please leave the two spaces between sentences intact.  Better to add it between the sentences that didn’t have it rather than remove it from the ones that do.

Reply:
> Please leave the two spaces between sentences intact. Better to add it between the sentences that didn’t have it rather than remove it from the ones that do.

Changed back, thx.